tests/components/admin_console/__snapshots__/custom_plugin_settings.test.jsx.snap

```diff
@@ -295,7 +295,7 @@ exports[`components/admin_console/CustomPluginSettings should match snapshot wit
         placeholder="e.g. some setting"
         setByEnv={false}
         type="input"
-        value="setting_default"
+        value="fsdsdg"
```

Comment:
The snapshot was wrong but in the PR that changed the admin console there was a ton of updates so we missed this one becoming broken